### DISCUSSION TITLE
Android app no audio

### DISCUSSION THREAD

**Discussion by napalmftw** [chuot/rdio-scanner#383] [General]
EDIT:

Ok, time to eat some humble pie.  I thought I had ffmpeg installed but I didn't.  At least not correctly.  I installed rdio-scanner in c:\ instead of leaving it in my downloads folder, and installed ffmpeg according to the directions in the pdf with rdio-scanner and now the app works.

The web interface to my server works fine with wave encoding and with h264 m4a (aside from it timing out and giving me No-Link after a while of background working).  The native android app gives me no audio.  I can see the server doing its part, and I can see the list of saved calls but when I click on them on my phone I get a loading icon and then nothing.  No playback.

I've looked at permissions, but nothing is denied for it.

Samsung Galaxy S22 Ultra and everything is updated.

**Comment by MrJordanRoth** [comment]
I had to disable audio conversion on the web interface after I installed FFMPEG. I experienced the same audio issue and a good test if you can go back the the playback and listen to something before you installed it you may see a difference.  My issue was with an IPad, bur very similar to yours.
